keps/sig-api-machinery/2885-server-side-unknown-field-validation/kep.yaml

```diff
@@ -18,26 +18,26 @@ see-also:
 replaces:
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.25"
+latest-milestone: "v1.27"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.23"
   beta: "v1.25"
-  stable: "v1.26"
+  stable: "v1.27"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
   - name: ServerSideFieldValidation
     components:
       - kube-apiserver
-disable-supported: true
+disable-supported: false
```

Comment:
I *think* this meant "could the gate be disabled during feature development", not "can the gate be disabled when locked to true at GA"

Reply:
Gotcha, I've reverted this for now.

It looks like based on this [one example](https://github.com/kubernetes/kubernetes/commit/a3d00c15b6464ea39e9599ad0863c51d6550bd4d) we remove the feature gate entirely once it's GA. This one did it three releases after going GA, but I can't tell if that wait is required, or if we can just remove the gate in the release that it goes GA. Do you know?

If so, I imagine we can just keep the gate in the kep.yaml file but adda comment that the gate was removed in release 1.x like I've done here?